packages/backend/src/core/activity/DailyTransactionCountView.ts

```diff
@@ -0,0 +1,73 @@
+import { Logger, TaskQueue } from '@l2beat/common'
+import { ProjectId, UnixTime } from '@l2beat/types'
+
+import { DailyTransactionCountViewRepository } from '../../peripherals/database/activity-v2/DailyTransactionCountRepository'
+import { Clock } from '../Clock'
+import { ALL_PROCESSED_EVENT, SequenceProcessor } from '../SequenceProcessor'
+import { decideAboutYesterday } from './decideAboutYesterday'
+import { fillMissingCounts } from './fillMissingCounts'
+import { groupByProjectId } from './groupByProjectId'
+import { DailyTransactionCount } from './types'
+
+export class DailyTransactionCountView {
```

Comment:
`DailyTransactionCountView` -> `DailyTransactionCountService`.

This more generic name will account for the fact that this class refreshes the views and also allows querying the data.

I think the relationship between this, ActivityModuleV2 and the ActivityV2Controller could be improved, but I think that this can definitely be done in a future PR.